### ISSUE TITLE
[fiddle] + dark mode

### ISSUE THREAD

**Issue by tomByrer** [BuilderIO/mitosis#68]
Neat tool!

The lack of dark mode is a bit of A11y issue for me; too much white screens can aggravate my concussion. I had to install a "Dark Mode" browser exertion; the fiddle actually looks better with it on to me!  (I'd only make the line numbers more transparent/less bright)  Unfortunately this doesn't work in VSCode.
![image](https://user-images.githubusercontent.com/1308419/109423911-0678a000-79a7-11eb-9cf9-377553343740.png)

(Side note; I'm not totally motion sensitive, but FPS are a no-no now :/  Some motion is fine when I expect it, but constantly animating something like a slider or pulsing menu highlight is a bit more frustrating now.)

Also, I find there isn't enough contrast &/or line thickness for your current white mode.

**Comment by steve8708** [comment]
thanks @tomByrer - this is a huge help. we've got a whole sweep to allow theme settings throughout our app (dark mode, higher contract, animation reduction/removal, etc) on our roadmap for later this year and we'll happily incorporate your feedback here!
